src/Model/BlogCategory.php

```diff
@@ -33,6 +33,13 @@ class BlogCategory extends DataObject implements CategorisationObject
      */
     private static $table_name = 'BlogCategory';
 
+    /**
+     * @config
+     *
+     * @var boolean
+     */
+    private static $allow_urlsegment_multibyte = true;
```

Comment:
```suggestion
    private static bool $allow_urlsegment_multibyte = true;
```
- `@config` is implied and is likely to be removed in a future major release anyway.
- We prefer to explicitly type-hint new API, even config. It's not enforced currently but could be in the future, and typehinting now will reduce the effort to have full typehinting across the board in the future.

Reply:
Please do this for `BlogTag` as well.